zookeeper-server/src/main/java/org/apache/zookeeper/server/ZKDatabase.java

```diff
@@ -333,6 +367,11 @@ public void addCommittedProposal(Request request) {
             p.packet = pp;
             p.request = request;
             committedLog.add(p);
+            int proposalSize = data != null ? data.length * 2 : 0;
```

Comment:
Maybe we can add a method to compute the size of a proposal, since we're doing it from 2 places. 

Also, why is the size doubled?